### ISSUE TITLE
Can I play a partial video in IINA?

### ISSUE THREAD

**Issue by ondialogue** [iina/iina#3711]
I want to play a internal video, not the whole video. How can I use IINA to play it?

**Comment by low-batt** [comment]
Yes, you can play part of a video. I'm not quite sure what you want to do. I'm assuming you don't want to use the slider on the on screen controler to move to the location from which you want to start playback?

IINA is a graphical user interface on top of the [mpv](https://mpv.io/) media player for the command line. IINA embeds `libmpv` for video and audio support along with libraries from [FFmpeg](https://ffmpeg.org/). Much of the full power of mpv is available in IINA. 

The Playback Control section of the [mpv manual](https://mpv.io/manual/stable/#playback-control) has examples like:
> **--start=30 --end=40**
    Seeks to 30 seconds, plays 10 seconds, and exits.
**--start=-3:20 --length=10**
    Seeks to 3 minutes and 20 seconds before the end of the file, plays 10 seconds, and exits.
**--start='#2' --end='#4'**
    Plays chapters 2 and 3, and exits.
	
After I disabled `Resume last playback position` in IINA preferences I was able to play 10 seconds of the movie starting 30 seconds in doing this in terminal:
```bash
low-batt@gag ~$ low-batt@gag ~$ /Applications/IINA.app/Contents/MacOS/iina-cli --mpv-start=0:30 --mpv-end=0:40 ~/Movies/earth.mp4 
```

**Comment by ondialogue** [comment]
Thanks!  This method can play one section of a video.  How can I play more than one sections?

**Comment by lhc70000** [comment]
I don't think there's a feature to allow you to play multiple discrete segments; that's a rare use case. May I ask under what situation you need this function?

With a custom mpv lua script or the plugin system (unfortunately you may need to wait for perhaps 2-3 months for the plugin system), it might be possible.
